src/ray/gcs/gcs_server/gcs_placement_group_manager.h

```diff
@@ -75,6 +75,9 @@ class GcsPlacementGroup {
   /// Get the bundles of this placement_group (including unplaced).
   std::vector<std::shared_ptr<BundleSpecification>> GetBundles() const;
 
+  /// Get the size of bundles (including unplaced).
+  size_t GetBundleSize() const;
```

Comment:
What does size mean? NumBundlesTotal()? Also, should not be size_t.

Reply:
This method is removed.